web/app/view/DevicesController.js

```diff
@@ -87,7 +89,12 @@ Ext.define('Traccar.view.DevicesController', {
                 if (btn === 'yes') {
                     store = Ext.getStore('Devices');
                     store.remove(device);
-                    store.sync();
+                    store.sync({
+                        failure: function (batch) {
```

Comment:
Should we handle it for other service calls as well? Then I guess it should be shared somewhere.